installer/pyinstaller/build-linux.sh

```diff
@@ -47,15 +48,32 @@ echo "Installing PyInstaller"
 
 echo "Building Binary"
 cd src
+if ! [ "$nightly_build" = "" ]; then
+    echo "Updating samcli.spec with nightly build"
+    sed -i "s/'sam'/'sam-nightly'/g" installer/pyinstaller/samcli.spec
```

Comment:
`-i` as-is won't work on BSD `sed` AFAIK. If it doesn't matter if we create files, `-i.bak` should work on both BSD and GNU.

Reply:
done.